lpa-submissions-e2e-tests/cypress/integration/accuracyAppellantsSubmission.feature

```diff
@@ -3,34 +3,34 @@ Feature: Accuracy of the Apellants Submission
   As a beta LPA I want to respond whether the information from the appellant accurately reflects the original planning application
```

Comment:
Should e2e testing be part of a view feature story?

Reply:
That's outside the scope of the ticket - AS-2 is just the question part of that ticket